src/App.tsx

```diff
@@ -1,8 +1,36 @@
+import {
+  Routes, Route, Link, Navigate,
+} from 'react-router-dom';
 import './App.scss';
+import { NotFoundPage } from './components/NotFoundPage';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
 
 const App = () => (
   <div className="App">
-    <h1>People table</h1>
+    <header>
+      <nav className="navigation">
+        <Link
+          className="navigation__link"
+          to="/"
+        >
+          Home
+        </Link>
+        <Link
+          className="navigation__link"
+          to="/people"
+        >
+          People
+        </Link>
+      </nav>
+    </header>
+
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+      <Route path="/people" element={<PeoplePage />} />
+      <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
add `replace` prop
here is why – https://reach.tech/router/api/navigate (at the bottom of the article)